### ISSUE TITLE
Settings seems to not be taken into account

### ISSUE THREAD

**Issue by guenogu** [wooorm/remark-lint#54]
Given the following files

**test.md**

    *emphasized* topic
    
    ```bash
    $ cd /path/
    
    $ source .env
    ```

**validate.md.sh**

    #!/bin/bash
    set -e

    FILES=$(find . -name "*.md" -not -path "*/node_modules/*" -type f)

    for f in $FILES
    do
        cat $f | remark -u remark-lint --frail --no-stdout --quiet --file-path $f
    done

    exit $?

**.remarkrc.js**

    module.exports = {
      "plugins": {
        "remark-lint": {
          "emphasis-marker": "_",
          "no-shell-dollars": false
        }
      },
      "settings": {
        "commonmark": true
      }
    };

**package.json**

    {
      "name": "repro-remark-lint",
      "version": "1.0.0",
      "license": "MIT",
      "private": true,
      "engines": {
        "node": "4.x"
      },
      "devDependencies": {
        "remark": "^4.2.0",
        "remark-lint": "^3.1.0"
      },
      "scripts": {
        "lint:md": "bash validate.md.sh"
      }
    }

The following commands issue some wrong errors:

```
$ npm install
$ npm run lint:md

test.md
    3:1-7:4  warning  Do not use dollar signs before shell-commands  no-shell-dollars

‼ 1 warning
```

Where I was rather expecting an error related to the wrong emphasis marker.

It looks like the configuration may not be taken into account.

**Comment by wooorm** [comment]
Could you provide the output of running `npm run lint:md`, if the `lint:md` task is replaced with `DEBUG="*" bash validate.md.sh`? Thanks!

**Comment by guenogu** [comment]
Here you are

```

Mon, 04 Apr 2016 16:09:14 GMT remark:cli:ignore Constructing ignore for `D:\Download\remark-lint-repro`
Mon, 04 Apr 2016 16:09:14 GMT remark:cli:file-set-pipeline:stdin Checking stdin
Mon, 04 Apr 2016 16:09:14 GMT remark:cli:file-set-pipeline:stdin Reading from stdin
Mon, 04 Apr 2016 16:09:14 GMT remark:cli:file-set-pipeline:stdin Read from stdin
Mon, 04 Apr 2016 16:09:14 GMT remark:cli:configuration Constructing configuration for `test.md`
Mon, 04 Apr 2016 16:09:14 GMT remark:cli:configuration Using D:\Download\remark-lint-repro\package.json
Mon, 04 Apr 2016 16:09:14 GMT remark:cli:configuration Using D:\Download\remark-lint-repro\.remarkrc.js
Mon, 04 Apr 2016 16:09:14 GMT remark:cli:file-pipeline:configure Setting output `undefined`
Mon, 04 Apr 2016 16:09:14 GMT remark:cli:file-pipeline:configure Using settings `{"position":true,"gfm":true,"yaml":true,"commonmark":true,"footnotes":false,"pedantic":false,"breaks":false,"entities":"false","setext":false,"closeAtx":false,"looseTable":false,"spacedTable":true,"incrementListMarker":true,"fences":false,"fence":"`","bullet":"-","listItemIndent":"tab","rule":"*","ruleSpaces":true,"ruleRepetition":3,"strong":"*","emphasis":"_"}`
Mon, 04 Apr 2016 16:09:14 GMT remark:cli:file-pipeline:configure Using plug-ins `["remark-lint"]`
Mon, 04 Apr 2016 16:09:14 GMT remark:cli:file-pipeline:configure Using plug-in `remark-lint` at `D:\Download\remark-lint-repro\node_modules\remark-lint`
Mon, 04 Apr 2016 16:09:14 GMT remark:cli:file-pipeline:configure Applying options `undefined` to `remark-lint`
Mon, 04 Apr 2016 16:09:14 GMT remark:cli:file-pipeline:configure Using `0` injected plugins
Mon, 04 Apr 2016 16:09:14 GMT remark:cli:file-pipeline:parse Parsing document
Mon, 04 Apr 2016 16:09:14 GMT remark:cli:file-pipeline:parse Parsed document
Mon, 04 Apr 2016 16:09:14 GMT remark:cli:file-pipeline:transform Transforming document test.md
Mon, 04 Apr 2016 16:09:14 GMT remark:cli:file-pipeline:transform Transformed document (error: null)
Mon, 04 Apr 2016 16:09:14 GMT remark:cli:file-pipeline:queue Queueing test.md
Mon, 04 Apr 2016 16:09:14 GMT remark:cli:file-pipeline:queue Can flush for `test.md`
Mon, 04 Apr 2016 16:09:14 GMT remark:cli:file-pipeline:queue Flushing
Mon, 04 Apr 2016 16:09:14 GMT remark:cli:file-pipeline:stringify Compiling document
Mon, 04 Apr 2016 16:09:14 GMT remark:cli:file-pipeline:stringify Compiled document to md
Mon, 04 Apr 2016 16:09:14 GMT remark:cli:file-pipeline:copy Not copying
Mon, 04 Apr 2016 16:09:14 GMT remark:cli:file-pipeline:stdout Ignoring writing to standard out
Mon, 04 Apr 2016 16:09:14 GMT remark:cli:file-pipeline:file-system Ignoring writing to file-system
test.md
    3:1-7:4  warning  Do not use dollar signs before shell-commands  no-shell-dollars

```

**Comment by wooorm** [comment]
Found it! You’re resetting the setting for `remark-lint`, which was previously set by `.remarkrc.js`, in the `remark -u remark-lint`. Drop the `-u remark-lint` part, it’s duplicate. See this part in the readme: _We now run the below command without the `-u remark-lint` since our `.remarkrc` includes the lint plugin._

**Comment by fermaem** [comment]
:cool: :bangbang: That did the trick. Thanks for the support :heart_eyes:
